### PR TITLE
Solving the tasks for Module #1

### PR DESCRIPTION
**Pull request by NatalliaBileichyk**
Module #1 includes tasks:

1. The grade of the student
2. Euclidean algorithm
3. Fibonacci sequence

### CODE REVIEW

File: module-1/fibonacci.js [hunk 1]
```diff
@@ -15,6 +15,20 @@ function fibonacci(n) {
      * Also take into consideration the documentation of the function!
      */
     // PLACE YOUR CODE BETWEEN THIS...
+    
+nThFibonacci = 0;
+let firstSum = 0;
+let secondSum = 1;
+if (n >= 0) {
```
Comment (rakur) [inline module-1/fibonacci.js]: It is good practice to make a check at the beginning to see if we have valid values, before even initiating the variables.

Comment (NatalliaBileichyk) [reply module-1/fibonacci.js]: I agree. I correct this line.

File: module-1/Task1(diagonalDifference).js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+function diagonalDifference(arr) {
+    // Write your code here
+const n = arr[0].length;
+
+let primaryDiagonal = 0;
+let secondaryDiagonal = 0;
+let i = 0;
```
Comment (rakur) [inline module-1/Task1(diagonalDifference).js]: Loop variables can be declared within the loop itself, limiting their scope

File: module-1/Task1(diagonalDifference).js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+let j = 0;
```
Comment (rakur) [inline module-1/Task1(diagonalDifference).js:8]: Do you need this 2 variables? Can you think of a formula with only the loop variable and the array length?

File: module-1/Task2(MinMax).js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+let arr = [7, 69, 2, 221, 8974]
+
+let sum = arr.reduce((total, value) => (total + value));
+let sumWithoutIndex = arr.map(element => sum - element);
+
+let minSum = sumWithoutIndex[0];
+for (let i of sumWithoutIndex) {
+    if (i > minSum) {continue;} else {minSum = i;};
```
Comment (rakur) [inline module-1/Task2(MinMax).js]: If you swap around the condition, you won't need the else part

**Review by rakur — APPROVED** [review]

File: module-1/Task2(MinMax).js (hunk 4 above)
Comment (NatalliaBileichyk) [reply module-1/Task2(MinMax).js]: Done

File: module-1/Task1(diagonalDifference).js (hunk 2 above)
Comment (NatalliaBileichyk) [reply module-1/Task1(diagonalDifference).js]: Done

**Pull request merged**